.github/workflows/upmerge.yaml

```diff
@@ -62,7 +62,9 @@ jobs:
           export SOURCE_BRANCH=$(basename ${{ github.ref }})
           echo "Upmerging samples from $SOURCE_BRANCH to edge"
           git fetch origin $SOURCE_BRANCH
-          git merge -m "Upmerge to edge" origin/$SOURCE_BRANCH
+          git merge --no-commit origin/$SOURCE_BRANCH
+          git checkout edge -- bicepconfig.json
+          git commit -m "Upmerge to edge"
```

Comment:
Let's add/update comment to include the reason why bicepconfig.json is excluded from merge

Reply:
Updated